### PR TITLE
Adding Controllers

### PR DESCRIPTION
**Pull request by SaveryIV**
In this section, I have completed the following activities:

1. Did NOT use rails to generate scaffold command. The point of this project is to show you how to set up controllers by myself.

2. Ensured that my app handles the URLs used in the wireframes:

3. For each URL, I created the following items:

     - A route
     - An action in the correct controller
     - A view file

4. View files should include only placeholders written in plain HTML, e.g: Here is a list of posts for a given user.

5. Ensured that I added code to handle only the 4 required URLs.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @SaveryIV, it would be nice to remove the live demo section for now till  you have a deployed link, also it would  be nice to update the content of your readme as they are talking about your capstone project,  please this is a comment not a review 👍🏿

**Comment by SaveryIV** [comment]
> Hi @SaveryIV, it would be nice to remove the live demo section for now till you have a deployed link, please this is a comment not a review 👍🏿

Okey i remove that

**Comment by SaveryIV** [comment]
@prolajumokeoni  is ready 👍

**Comment by prolajumokeoni** [comment]
> also it would be nice to update the content of your readme as they are talking about your capstone project, please this is a comment not a review 👍🏿

@SaveryIV  It would be nice to update the content of your readme as they are talking about your catalog of things capstone project, Kindly check the key and future features section in your readme and update it, please this is a comment not a review 👍🏿

**Comment by SaveryIV** [comment]
> > also it would be nice to update the content of your readme as they are talking about your capstone project, please this is a comment not a review 👍🏿
> 
> @SaveryIV It would be nice to update the content of your readme as they are talking about your catalog of things capstone project, Kindly check the key and future features section in your readme and update it, please this is a comment not a review 👍🏿

Nice i fix it @prolajumokeoni  👍

**Comment by SaveryIV** [comment]
@prolajumokeoni im approved?

File: README.md [hunk 1]
```diff
@@ -49,20 +49,9 @@ A Ruby blog app is a web application that allows users to create, edit, and publ
...
-- **Ability to track item status**
-- **Ability to generate reports**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-## 🚀 Live Demo <a name="live-demo"></a>
-
-The Live Demo of this Capstone Project is available now on YouTube.
-
```
Comment (prolajumokeoni) [inline README.md:64]: - [x] Please update the installation and setup section for your readme to talk about your current project 👍🏿 

- [ ] Please remove the live demo section from your project for now till you have a deployed link, this makes your readme more professional 👍🏿 


test

- [x] Kindly update your future features section, content it is still referring to my catatlog of things project 👍🏿 

ttt

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @SaveryIV 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Controllers,  Views and Routes setup correctly 👏


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by prolajumokeoni** [comment]
> @prolajumokeoni im approved?

Hi not yet reading through your project readme I see some places you are still referring to my catalog of things project, please go through your readme and update requested changes  👍🏿

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @SaveryIV 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**